src/Viewer.tsx

```diff
@@ -514,7 +528,8 @@ function Viewer(): ReactElement {
       }
 
       setCollection(newCollection);
-      const datasetResult = await newCollection.tryLoadDataset(datasetKey);
+      setDatasetLoadProgress(null);
```

Comment:
Question unrelated to this change: I'm noticing this async function is defined within a `useEffect` callback, then the only other thing the callback does is call this function. Could the callback itself not be async?

Reply:
Ah, `useEffect` does not allow async callbacks.

```
Argument of type '() => Promise<void>' is not assignable to parameter of type 'EffectCallback'.
```